src/3rd_party/apache-log4cxx-0.10.0/autom4te.cache/requests

```diff
@@ -15,63 +15,63 @@
                         'configure.in'
                       ],
                       {
-                        'AM_MAKEFILE_INCLUDE' => 1,
+                        'AM_PROG_FC_C_O' => 1,
```

Comment:
@LitvinenkoIra I suppose this files are autogenerated and should not be committed